fdbserver/workloads/ClientTransactionProfileCorrectness.actor.cpp

```diff
@@ -41,51 +124,37 @@ bool checkTxInfoEntryFormat(BinaryReader &reader) {
 		{
 		case FdbClientLogEvents::GET_VERSION_LATENCY:
 		{
```

Comment:
Now these "{" and "}" from here through line 159 can be removed.

Reply:
Ok.